sentry_sdk/integrations/django/signals_handlers.py

```diff
@@ -13,19 +13,32 @@
     from typing import List
 
 
+def _get_receiver_name(receiver):
+    # type: (Callable[..., Any]) -> str
+    name = ""
+
+    if hasattr(receiver, "__qualname__"):
+        name += receiver.__qualname__
+    elif hasattr(receiver, "__name__"):  # Python 2.7 has no __qualname__
+        name += receiver.__name__
+
+    if (
+        name == ""
+    ):  # certain functions (like partials) dont have a name so return the string representation
+        return str(receiver)
+
+    if hasattr(receiver, "__module__"):  # prepend with module, if there is one
```

Comment:
qualname already has the module prefix right?

Reply:
No. It is only the fully qualified name of the class itself (with dots if is a nested class) but it does not include the python module.

Reply:
ah gotcha, thx